.gitignore

```diff
@@ -278,12 +278,15 @@ storybook-static
 **/__snapshots__
 .env*
 !.env
-/test-results/
-/playwright-report/
-/playwright/.cache/
 
 # Sentry Auth Token
 .sentryclirc
 
 # Local Netlify folder
 .netlify
+
+# Playwright
```

Comment:
One of the test artifacts folders was missing. I also moved this to it's own section to explain what they're ignored for.